security/pkg/pki/ra/k8s_ra_test.go

```diff
@@ -179,6 +179,7 @@ func createFakeCsr(t *testing.T) []byte {
 
 func initFakeKubeClient(t test.Failer, certificate []byte) kube.CLIClient {
 	client := kube.NewFakeClient()
+	client.RunAndWait(test.NewStop(t))
```

Comment:
This PR basically implicitly enforces a new invariant: and clients created must eventually be Run(), or they will leak a goroutine due to the queue usage.

This could be fixed by make a 1-10 year PR in client-go to avoid leaking goroutines, but since it only impacts one test... probably not worth it.